datafusion/physical-expr/src/aggregate/first_last.rs

```diff
@@ -44,6 +44,7 @@ pub struct FirstValue {
     expr: Arc<dyn PhysicalExpr>,
     ordering_req: LexOrdering,
     requirement_satisfied: bool,
+    ignore_null: bool,
```

Comment:
a very minor nit is that some parts of this PR use `ignore_null` and some parts use `ignore_nulls` (with an `s`). It might be nice to make them uniform

Reply:
I have change all of them to `ignore_nulls`. Thanks for catching this!